docs/4.2. Behaviour - Querying.md

```diff
@@ -2,63 +2,63 @@
 
-`authorization` Indicates whether the WebSocket connection requires authorization in order to connect or not. Use of authorization is likely to be a deployment decision and be the same for all subscriptions exposed from a single Query API, and for the RESTful API itself. Query API clients MAY choose to specify a requested value for this attribute, but in most circumstances this will result in a 400 response code if the Query API is operating in the opposite mode.
+`authorization` Indicates whether the WebSocket connection requires authorization in order to connect. Use of authorization is likely to be a deployment decision and be the same for all subscriptions exposed from a single Query API, and for the RESTful API itself. Query API clients MAY specify a requested value for this attribute, but in most circumstances this will result in a 400 response code if the Query API is operating in the opposite mode.
 
-## Creating a Websocket subscription
+## Creating a WebSocket subscription
 
-In order to connect to a websocket, a client must first request a subscription of a particular type and with particular query parameters by performing a POST as defined in the [Query API](../APIs/QueryAPI.raml) subscriptions documentation. Use of GET requests to find existing suitable websocket connections is strongly discouraged.
+In order to connect to a WebSocket, a client MUST first request a subscription of a particular type and with particular query parameters by performing a POST as defined in the [Query API](../APIs/QueryAPI.raml) subscriptions documentation. Use of GET requests to find existing suitable WebSocket connections is strongly discouraged.
 
-There is no mandated URL base path for servers to use when creating websocket connections. Instead clients should observe the value of 'ws_href' which is returned by their subscription request in order to identify what to connect to.
+There is no mandated URL base path for servers to use when creating WebSocket connections. Instead clients SHOULD observe the value of 'ws_href' which is returned by their subscription request in order to identify what to connect to.
 
-Websockets created for specific clients will be cleaned up automatically if they disconnect.
+WebSockets created for specific clients will be cleaned up automatically if they disconnect.
 
-Connection upgrade procedures permitting an existing HTTP query to transition to a websocket are not currently supported, but may be in the future.
+Connection upgrade procedures permitting an existing HTTP query to transition to a WebSocket are not currently supported, but could be in the future.
```

Comment:
Maybe change "supported" to indicate "covered by this specification" better? Although, see next section, first para!